### DISCUSSION TITLE
Is there a way to instruct Bun to substitute Uint8Array for Buffer when using bun build?

### DISCUSSION THREAD

**Discussion by guest271314** [oven-sh/bun#8001] [Q&A]
Is there a way to instruct Bun to substitute `Uint8Array` for `Buffer` when using `bun build`?

**Comment by Jarred-Sumner** [comment]
Where? I don't think we return a `Uint8Array` anywhere

**Reply by guest271314** [reply]
I mean the literal use of Node.js-specific `Buffer` in the bundled code.
